lib/python/Screens/Information.py

```diff
@@ -1411,7 +1411,7 @@ def findPackageRevision(package, packageList):
 		info.append(formatLine("S", _("Driver and kernel information")))
 		if self.extraSpacing:
 			info.append("")
-		info.append(formatLine("P1", _("Drivers version"), formatDate(about.getDriverInstalledDate())))
+		info.append(formatLine("P1", _("Drivers version"), formatDate(BoxInfo.getItem("driversdate"))))
```

Comment:
Does this mean that "about.getDriverInstalledDate()" can also now be deleted?

Reply:
driversdate on enigma-kernel module fixed by @atvcaptain 20 days ago. working as well
https://github.com/oe-alliance/oe-alliance-core/commit/f5e9e7b5bb0e0ad5466fcb347dd20a97548154e2